Extensions/PrimitiveDrawing/shapepainterruntimeobject-pixi-renderer.ts

```diff
@@ -485,8 +486,35 @@ namespace gdjs {
       point[1] = position.y;
       return point;
     }
+    applyAntialiasing(): void {
+      if (this._object.isAntialiasingOn()) {
+        let antialiasingFilter = new PIXI.filters.FXAAFilter();
+        antialiasingFilter.enabled = true;
+        switch (this._object.getAntialiasingQuality()) {
+          case 2:
+            antialiasingFilter.multisample = PIXI.MSAA_QUALITY.LOW;
+            break;
+          case 4:
+            antialiasingFilter.multisample = PIXI.MSAA_QUALITY.MEDIUM;
+            break;
+          case 8:
+            antialiasingFilter.multisample = PIXI.MSAA_QUALITY.HIGH;
+            break;
+          default:
+            //Should never be executed but if it's not a valid value, set the quality to LOW.
+            antialiasingFilter.multisample = PIXI.MSAA_QUALITY.LOW;
+
+            break;
+        }
```

Comment:
If you use a `stringWithSelector` as suggested by @D8H, you can simplify this whole code block to this:
```suggestion
        antialiasingFilter.multisample = PIXI.MSAA_QUALITY[this._object.getAntialiasingQuality()] || PIXI.MSAA_QUALITY.LOW;
```